lib/bolt/transport/winrm/connection.rb

```diff
@@ -156,6 +156,7 @@ def write_remote_file(source, destination)
         end
 
         def write_remote_file_winrm(source, destination)
+          @logger.debug { "Uploading #{source}, to #{destination}" }
```

Comment:
It seems like there's a few other methods in this file that need this - I think:
```
lucy ~/githubs/bolt (master) $ ag -r "def write" lib/bolt/transport/winrm/connection.rb 
150:        def write_remote_file(source, destination)
158:        def write_remote_file_winrm(source, destination)
165:        def write_remote_file_smb(source, destination)
218:        def write_remote_executable(dir, file, filename = nil)
226:        def write_executable_from_content(dir, content, filename)
280:        def write_remote_file_smb_recursive(tree, source, dest)
```

Reply:
oh, I think write_remote_file should cover all those. I'm not sure what `write_executable_from_content` is used for (actually for both ssh and winrm i dont think that method is used).